tests/persistent_volumes_test.go

```diff
@@ -78,3 +85,89 @@ func TestPersistence(t *testing.T) {
 	// Cleanup
 	removeDeployment(deploymentClient, deploymentTemplate.GetName(), k8sNameSpace)
 }
+
+func TestPVCResize(t *testing.T) {
```

Comment:
To test properly, you should add a test for shrinking volume size (which should not happen and have an event saying so)